interpreter/cling/include/cling/Interpreter/Value.h

```diff
@@ -263,11 +263,7 @@ namespace cling {
 #ifndef NDEBUG
 #define _STRINGIFY(x) #x
 #define STRINGIFY(x) _STRINGIFY(x)
-    // FIXME: Uncomment and debug the various type mismatches.
-    //#define ASSERT_TYPE_MISMATCH(name) AssertTypeMismatch(STRINGIFY(name))
-    #define ASSERT_TYPE_MISMATCH(name)
-#undef STRINGIFY
-#undef _STRINGIFY
+    #define ASSERT_TYPE_MISMATCH(name) AssertTypeMismatch(STRINGIFY(name))
```

Comment:
Would it make sense to undefine this later in this file?

Reply:
Sure, but too early to approve - we need to solve a few problems...

Reply:
Well, I trust you on those problems, so that doesn't stop me from approving! :-)